lib/portage/dbapi/vartree.py

```diff
@@ -4609,19 +4596,19 @@ def onerror(e):
             self._elog("eqawarn", "preinst", _merge_unicode_error(unicode_errors))
 
         if paths_with_newlines:
-            msg = []
-            msg.append(
+            msg = [
                 _(
                     "This package installs one or more files containing line ending characters:"
-                )
-            )
-            msg.append("")
-            paths_with_newlines.sort()
-            for f in paths_with_newlines:
-                msg.append("\t/%s" % (f.replace("\n", "\\n").replace("\r", "\\r")))
-            msg.append("")
-            msg.append(_("package %s NOT merged") % self.mycpv)
-            msg.append("")
+                ),
+                "",
+                *(
+                    "\t/{}".format(f.replace("\n", "\\n").replace("\r", "\\r"))
+                    for f in sorted(paths_with_newlines)
+                ),
+                "",
+                _("package {self.mycpv} NOT merged"),
+                "",
+            ]
```

Comment:
Same here.

Reply:
I'm not understanding what's being requested here.

Are you referring to how I used this generator for expanding into the list? And if so, would putting it in its own describable variable to replace this expression inside the list with suffice?
```python
*(
    "\t/{}".format(f.replace("\n", "\\n").replace("\r", "\\r"))
    for f in sorted(paths_with_newlines)
),
```